docker/openemr/flex-edge/autoconfig.sh

```diff
@@ -238,6 +238,23 @@ if [ -f /var/www/localhost/htdocs/auto_configure.php ] &&
 
     if [ -f /var/www/localhost/htdocs/openemr/package.json ]; then
         # install frontend dependencies (need unsafe-perm to run as root)
+        # IN ALPINE 3.14, there is an odd permission thing happening where need to give non-root ownership
+        #  to several places ('node_modules' and 'public') in flex environment that npm is accessing via:
+        #    'chown -R apache:1000 node_modules'
+        #    'chown -R apache:1000 ccdaservice/node_modules'
+        #    'chown -R apache:1000 public'
+        # WILL KEEP TRYING TO REMOVE THESE LINES IN THE FUTURE SINCE APPEARS TO LIKELY BE A FLEETING NPM BUG WITH --unsafe-perm SETTING
```

Comment:
The critical bug in npm appears to be that the `--unsafe-perm` setting is not working as it should. Below is a work around, which is testing well.